edifact-edi/2.2/modules/ROOT/edits/index.adoc

```diff
@@ -0,0 +1,107 @@
+= EDIFACT EDI Connector 2.2 - Mule 4
+:page-aliases: connectors::edifact/edifact-edi-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+Anypoint Connector for EDIFACT EDI (EDIFACT Connector) enables you to convert EDIFACT messages to and from DataWeave-compatible representations using lists and maps.
+
+The  xref:edifact-edi-versions.adoc[supported EDIFACT versions] are provided.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+To use EDIFACT EDI Connector, you must have:
+
+* Purchased a MuleSoft license for Anypoint B2B
+
+* Anypoint Studio 7.0 or later
+
+[NOTE]
```

Comment:
I probably commented on this being a Note in the previous edit. It seems too important to be a Note at the end of the prereqs and I wonder if there is not a prereq category for its content: could it be incorporated into "you must have" as "Memory sufficient to meet the requirement for..."?